assets/html/documenter.css

```diff
@@ -428,3 +432,98 @@ article section.docstring a.source-link {
     background-color: transparent;
     padding: 0;
 }
+
+@media only screen and (max-width: 768px) {
+    div#topbar {
+        display: block; /* is mobile */
+        position: fixed;
+        width: 100%;
+        height: 2.7em;
```

Comment:
I would make the `#topbar` a bit wider in the vertical direction.

If I may I would propose a way to set the vertical heights in the following way, which I think would be a bit more clear:

* Have `padding-top: 1em` and `padding-bottom: 1em` on the `#topbar`.
* Drop `margin-top` from `div#topbar span` and only keep `padding-right: 1em` for `div#topbar a.fa-bars`
* Have an explicit `font-size: 1.5em` on the `div#topbar a.fa-bars` (as opposed `x-large`) -- no other reason than to have an explicit number here.
* The navlink text will be smaller, so for the `#topbar` it should also be `height: 1.5em`.
* The total height of the `#topbar` will then be 3.5em, so we should also have a `padding-top: 3em` on the `article`, since the default 2em will not be enough.
* The headroom should also then probably be something like `top: -4em`.